packages/fluent/scss/core/_index.scss

```diff
@@ -46,7 +48,14 @@
     $kendo-loading-text: $kendo-loading-text !default,
     // Elevation
     $_default-elevation: $kendo-elevation,
-    $kendo-elevation: $kendo-elevation !default
+    $kendo-elevation: $kendo-elevation !default,
+    // Typography
+    $_default-font-sizes: $kendo-font-sizes,
```

Comment:
I don't think that the private variables should be added here, but test it please.